src/emmental/model.py

```diff
@@ -284,7 +287,7 @@ def _get_data_from_output_dict(
                 )
         # Handle output_dict is neither a list or dict, and index is (X, Y)
         elif int(index[1]) == 0:
-            return [output_dict[index[0]]][int(index[1])]
+            return output_dict[index[0]]
```

Comment:
Small nit but can you change the comment to be that output_dict's items are list/dicts. In the comments you say "if output_duct is list, tuple, or dict". This is a little confusing as output_dict is a dict. It's values are Any type.